utils/src/main/scala/za/co/absa/enceladus/utils/unicode/ParameterConversion.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.utils.unicode
+
+object ParameterConversion {
+
+  implicit class UnicodeValueString(s: String) {
+    val unicodeElement = """[uU]?\+?([0-9a-fA-F]{4})""".r
+
+    def includingUnicode: String = s match {
```

Comment:
Not sure where we stand now. Do we implement "no" in this match as well for empty string? @yruslan

Reply:
Looks good. Or I may miss something.
After a discussion yesterday we decided to go for "none" as the keyword for the empty string.
This case is handled in `NoneValueString`. If a string parameter does not support empty strings, but support Unicode, `includingUnicode` matcher will be used with `NoneValueString`. But we decided to support empty string for eny char parameter for now, even for the delimiter.

Reply:
My comment was before the discussion happened.